saleor/graphql/context.py

```diff
@@ -17,7 +17,7 @@
 def get_context_value(request: HttpRequest) -> SaleorContext:
     request = cast(SaleorContext, request)
     request.dataloaders = {}
-    request.is_mutation = False
+    request.is_mutation = getattr(request, "is_mutation", False)
```

Comment:
We should change this as well to replace is_mutation with allow_replica entirely:
```suggestion
    request.is_mutation = getattr(request, "allow_replica", True)
```